ieeeuoft/hackathon-template
# IEEE-152-re-fetch-hardware-when-product-overview-loads

## Overview

- Resolves #<[IEEE-152](https://ieeeuoft.atlassian.net/browse/IEEE-152)>
- Once the user opens Product Overview Sidebar, hardware details are re-fetched to show the most up to date data
- Created a redux thunk for re-fetching the data
- Created a Loader for the Product Overview component


## Unit Tests Created

- Tested the thunk getUpdatedHardwareDetails in the hardwareSlice.test.tsx for API operations
- Tested the ProductOverview component for displaying up to date data (Issue: failing testcases)



## Steps to QA
- Navigate to Inventory
- Click on an inventory item
- Should see the most up to date data
